Assets/Scripts/Localization/LocalizationTable.cs

```diff
@@ -59,7 +61,7 @@ public static void LoadLocalizationFile(string path)
          * </para>
          * </summary>
          */
-        public static void LoadLocalizationFile(string path, string localizationCode)
+        public static void LoadLocalizationFile(string path, string localizationName)
```

Comment:
I do not think that it is goot idea to change "key" of the dictionary to be name. There should be another way to transalate language code to readable format without this.
```currentLanguage``` is changed from outside only when selection of language changes. At this point you should map Name of language to it's key/code.